tests/test_migrators.py

```diff
@@ -466,35 +470,63 @@ def run_test_migration(
     tmpdir: str,
     should_filter: bool = False,
     make_body: bool = False,
+    recipe_version: int = 0,
 ):
+    tmpdir_p = Path(tmpdir)
     if mr_out:
         mr_out.update(bot_rerun=False)
 
-    Path(tmpdir).joinpath("meta.yaml").write_text(inp)
+    if recipe_version == 0:
+        tmpdir_p.joinpath("meta.yaml").write_text(inp)
+        recipe_dir = str(tmpdir_p)
+    else:
+        tmpdir_p.joinpath(".ci_support").mkdir()
+        tmpdir_p.joinpath("recipe", "recipe.yaml").write_text(inp)
+        (tmpdir_p / ".ci_support" / "linux_64_.yaml").write_text(
+            "target_platform: linux-64"
+        )
 
+        recipe_dir = str(tmpdir_p / "recipe")
```

Comment:
Let's be consistent here.

```suggestion
    elif recipe_version == 1:
        tmpdir_p.joinpath(".ci_support").mkdir()
        tmpdir_p.joinpath("recipe", "recipe.yaml").write_text(inp)
        (tmpdir_p / ".ci_support" / "linux_64_.yaml").write_text(
            "target_platform: linux-64"
        )

        recipe_dir = str(tmpdir_p / "recipe")
    else:
        raise ValueError(f"Unsupported recipe version: {recipe_version}")
```

Reply:
Alright! Applied suggestion.